.wp-env.json

```diff
@@ -15,6 +15,8 @@
 	"config": {
 		"ELEMENTOR_SHOW_HIDDEN_EXPERIMENTS": true,
 		"SCRIPT_DEBUG": false,
-		"WP_DEBUG": false
+		"WP_DEBUG": true,
+		"WP_DEBUG_LOG": true,
+		"WP_DEBUG_DISPLAY": true
```

Comment:
@KingYes You can revert this change if you want. It was only to debug the PlayWright issue.